Lib/test/test_compare.py

```diff
@@ -1,21 +1,27 @@
+"Test equality and order comparisons."
```

Comment:
Do docstrings support single quotes?

```suggestion
"""Test equality and order comparisons."""
```

Reply:
Docstrings are string literals (not f-strings) that are the first statement in a module, class, or function body.  The parser does not know what quotes and prefixes (other than f) were removed by the tokenizer.  Using `"` rather than `'` is simply a style convention.  I personally use `"` rather than `"""` for single line docstrings, but since the author of the module and the patch did otherwise elsewhere, I will switch this one.